hint_accountant/src/main.rs

```diff
@@ -46,6 +31,21 @@ struct Whitelist {
 }
 
 fn run() {
+    // We use the files in the cairo-lang repo, cloned from the latest version
+    let whitelist_paths = fs::read_dir("../cairo-lang/src/starkware/starknet/security/whitelists")
+        .expect("Failed to read whitelist directory from cairo-lang, did you forget to clone it?");
```

Comment:
lets add this tip in case we want to run it locally

```suggestion
        .expect("Failed to read whitelist directory from cairo-lang, did you forget to clone it? \ 
        run https://github.com/starkware-libs/cairo-lang.git");
```

Not sure if the suggestion will work :P

Reply:
I wouldn't add the command there as we don't know where we are running from and the relative path may be wrong (also cloning cairo-lang is pretty self-explainatory). We can add a makefile target that clones the repo and runs the hint accountant if we want to do the comparison locally.

Reply:
Nice, let add the makefile command